### PR TITLE
Support Base mutations for name, teaser, and body

### PR DESCRIPTION
**Pull request by solocommand**
Requires #127 merged first to prevent conflicts in the updateContentMutationHandler

### CODE REVIEW

**Comment by solocommand** [comment]
[Create GraphQL mutations to allow modification of company and event fields](https://trello.com/c/MNb1mVwT/101-create-graphql-mutations-to-allow-modification-of-company-and-event-fields)

File: services/graphql-server/src/graphql/definitions/platform/content/index.js [hunk 1]
```diff
@@ -382,18 +386,42 @@ input ContentPublishingMutationInput {
   unpublished: Date
 }
 
+input ContentNameMutationInput {
+  "The content ID"
+  id: Int!
+  "The name of the content"
+  value: String
+  "The mutation to target"
```
Comment (zarathustra323) [inline services/graphql-server/src/graphql/definitions/platform/content/index.js]: Update this to say `The mutation to target. The default/non-mutated value will be used when this is null or unset.`

File: services/graphql-server/src/graphql/definitions/platform/content/index.js [hunk 2]
```diff
@@ -382,18 +386,42 @@ input ContentPublishingMutationInput {
...
+  mutation: ContentMutation
+}
+
 input ContentBodyMutationInput {
   "The content ID"
   id: Int!
   "The body text for the content"
-  body: String
+  value: String
+  "The mutation to target"
```
Comment (zarathustra323) [inline services/graphql-server/src/graphql/definitions/platform/content/index.js]: Same here

File: services/graphql-server/src/graphql/definitions/platform/content/index.js [hunk 3]
```diff
@@ -382,18 +386,42 @@ input ContentPublishingMutationInput {
...
+  mutation: ContentMutation
 }
 
 input ContentTeaserMutationInput {
   "The content ID"
   id: Int!
   "The teaser/intro text for the content"
-  teaser: String
+  value: String
+  "The mutation to target"
```
Comment (zarathustra323) [inline services/graphql-server/src/graphql/definitions/platform/content/index.js]: Same here

**Review by zarathustra323 — CHANGES_REQUESTED** [review]
@solocommand lets get the other PRs resolved and merged and then re-base this to make sure all the changes are correct. it looks like the event mutation code bled into this PR

**Review by zarathustra323 — APPROVED** [review]

**Pull request merged**